libminifi/include/utils/StringUtils.h

```diff
@@ -88,7 +91,10 @@ class StringUtils {
    */
   static inline bool equalsIgnoreCase(const std::string &left, const std::string right) {
     if (left.length() == right.length()) {
-      return std::equal(right.begin(), right.end(), left.begin(), [](unsigned char lc, unsigned char rc) {return std::tolower(lc) == std::tolower(rc);});
```

Comment:
I noticed you changed a lot of files outside of core. Can you keep these at 200 characters so we avoid unnecessary change later?